chia/data_layer/data_layer.py

```diff
@@ -452,13 +452,23 @@ async def upload_files(self, tree_id: bytes32) -> None:
                         "diff_path": str(write_file_result.diff_tree),
                     }
                     for uploader in uploaders:
+                        self.log.info(f"Using uploader {uploader} for store {tree_id.hex()}")
                         async with aiohttp.ClientSession() as session:
                             async with session.post(uploader + "/upload", json=request_json) as response:
                                 res_json = await response.json()
                                 if not res_json["uploaded"]:
+                                    self.log.error(
+                                        f"Failed to upload files to {uploader} : {res_json} - will retry later"
+                                    )
                                     break  # todo this will retry all uploaders
+                                else:
+                                    self.log.info(
+                                        f"Uploaded files to {uploader} for store {tree_id.hex()} "
+                                        "generation {publish_generation}"
```

Comment:
I think this was meant.

```suggestion
                                        f"Uploaded files to {uploader} for store {tree_id.hex()} "
                                        f"generation {publish_generation}"
```